rerun_py/src/python_bridge.rs

```diff
@@ -541,7 +511,27 @@ fn send_mem_sink_as_default_blueprint(
 }
 
 #[pyfunction]
-#[pyo3(signature = (addr = None, flush_timeout_sec=rerun::default_flush_timeout().unwrap().as_secs_f32(), default_blueprint = None, recording = None))]
+#[pyo3(signature = (port = 9876, memory_limit = "75%".to_owned(), executable_name = "rerun".to_owned(), executable_path = None, extra_args = vec![]))]
+fn spawn(
+    port: u16,
+    memory_limit: String,
+    executable_name: String,
+    executable_path: Option<String>,
+    extra_args: Vec<String>,
+) -> PyResult<()> {
+    let spawn_opts = re_sdk::SpawnOptions {
+        port,
+        memory_limit,
+        executable_name,
+        executable_path,
+        extra_args,
+    };
+
+    re_sdk::spawn(&spawn_opts).map_err(|err| PyRuntimeError::new_err(err.to_string()))
```

Comment:
Instead of calling the low-level `re_sdk::spawn` (that just spawns the binary), we could be using `RecordingStream::spawn_opts` here.

That would unify the handling of:
* Existing server on that port
* Initial connection

i.e. it would remove those parts of `rerun_py/rerun_sdk/rerun/_spawn.py`